helios-testing/src/test/java/com/spotify/helios/testing/UdpProberTest.java

```diff
@@ -66,7 +66,7 @@ public void setup() {
     @After
     public void tearDown() {
       // The TemporaryJobs Rule above doesn't undeploy the job for some reason...
-      job.undeploy();
+      // temporaryJobs.deployer().undeploy(job.job(), job.hosts());
```

Comment:
is this meant to be commented out? just remove the method if it has no actual code in it